android/src/main/java/com/vydia/RNUploader/UploaderModule.kt

```diff
@@ -318,6 +318,16 @@ class UploaderModule(val reactContext: ReactApplicationContext) : ReactContextBa
     }
   }
 
+  @ReactMethod
+  fun addListener(type: String?) {
+      // Keep: Required for RN built in Event Emitter Calls.
+  }
+
+  @ReactMethod
+  fun removeListeners(type: Int?) {
+      // Keep: Required for RN built in Event Emitter Calls.
+  }
+
```

Comment:
Fixes this warning:
https://stackoverflow.com/questions/69538962/new-nativeeventemitter-was-called-with-a-non-null-argument-without-the-requir